tgui/packages/tgui-panel/reconnect.tsx

```diff
@@ -0,0 +1,33 @@
+import { useSelector } from 'common/redux';
+import { Button } from 'tgui/components';
+
+const initialState: {
+  url?: string,
+} = {
+  url: undefined,
+};
+
+export const reconnectReducer = (state = initialState, action) => {
+  const { type, payload } = action;
+
+  if (type === 'reconnect/sendServerUrl') {
+    return {
+      url: payload.url,
+    };
+  }
+
+  return state;
+};
+
+export const ReconnectButton = (props, context) => {
+  const { url } = useSelector(context, state => state.reconnect);
+
+  return url && (
+    <Button color="white" onClick={() => {
+      location.href = `byond://${url}`;
+      Byond.command('.quit');
+    }}>
+      Reconnect
```

Comment:
because this can destroy any VV or log windows admins or debugging coders have open, it should be clear what this does.

```suggestion
      Relaunch Game
```

My original idea was to add it as a new button.

Reply:
I'll add both.